node/core/provisioner/src/metrics.rs

```diff
@@ -134,6 +141,15 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			inherent_data_response_bitfields: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_provisioner_inherent_data_response_bitfields_sent",
+						"Number of ineherent bitfields sent in response to `ProvisionerMessage::RequestInherentData`.",
+					).buckets(vec![0.0, 10.0, 25.0, 50.0, 75.0, 100.0, 150.0, 200.0, 250.0, 300.0]),
```

Comment:
Not great, not terrible: this bucket configuration works for Versi (300 para validators) and Kusama (200).